spec/spec_helper.rb

```diff
@@ -1,5 +1,30 @@
 # frozen_string_literal: true
 
+# Simplecov needs to be loaded before we require `memo_wise` in order to
+# properly track all memo_wise files
+if Gem.loaded_specs.key?("codecov")
```

Comment:
Why remove the `ENV[CI] == "true"` check here? This will make it run locally when we run specs?

Reply:
I actually found it useful to run locally, so moved the `ENV["CI"]` check below to determine the output. When running locally, it now outputs to the `coverage` directory so we can open `coverage/index.html` and clearly view the results. I found this helpful for local dev, but let me know if you're thinking differently!